ui/page/main_page.go

```diff
@@ -511,10 +524,39 @@ func (mp *MainPage) LayoutTopBar(gtx layout.Context) layout.Dimensions {
 											})
 									}),
 									layout.Rigid(func(gtx C) D {
+										if mp.hideBalance.Changed() {
+											mp.hide = !mp.hide
+										}
 										return layout.Center.Layout(gtx, func(gtx C) D {
-											return components.LayoutBalance(gtx, mp.Load, mp.totalBalance.String())
+											if mp.hide {
+												hidenText := mp.Theme.Label(values.TextSize24, "********")
+												return hidenText.Layout(gtx)
+											} else {
+												return components.LayoutBalance(gtx, mp.Load, mp.totalBalance.String())
+											}
```

Comment:
no need for the else condition as the return.
```suggestion
											} 
												return components.LayoutBalance(gtx, mp.Load, mp.totalBalance.String())
											
```